demo/hello.ts

```diff
@@ -385,5 +387,5 @@ export class HelloApp {
 })
 export class FormlyDemoModule {
 }
-
+enableProdMode();
```

Comment:
That will make sure that our demo will run in prod mode as well, I made some changes which were not running in prod mode, so having this could help us develop for production apps.